.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 4,
+  "semi": true,
+  "singleQuote": true,
+  "useTabs": true
+}
```

Comment:
I suspect this file isn't getting detected by the `format` script in `package.json`. Otherwise I think it should get a new line added at the end.

![Screenshot of the file ending with a red ⛔](https://user-images.githubusercontent.com/3335181/170187604-7f4bf8b8-217b-4c58-a218-407d5ff706fa.png)

Would this file get included with the change I suggested to remove include patterns? If not, does adding a `.json` extension work?